common/cmake/devices.cmake

```diff
@@ -1117,7 +1118,7 @@ function(ADD_FPGA_TARGET)
 
     add_custom_command(
       OUTPUT ${OUT_IO}
-      DEPENDS ${IO_DEPS}
+      DEPENDS ${IO_DEPS} ${OUT_NET}
```

Comment:
Should be `append_file_dependency(IO_DEPS ${OUT_NET_REL})` on IO_DEPS, it is insufficient to depend on the file directly.